charts/dex.yaml

```diff
@@ -0,0 +1,102 @@
+image: '{%- endraw -%}{{ build_image_name(\"dex\", False) }}{%- raw -%}'
+
+nodeSelector:
+  node-role.kubernetes.io/master: ''
+
+tolerations:
+  - key: "node-role.kubernetes.io/bootstrap"
+    operator: "Exists"
+    effect: "NoSchedule"
+  - key: "node-role.kubernetes.io/master"
+    operator: "Exists"
+    effect: "NoSchedule"
+  - key: "node-role.kubernetes.io/infra"
+    operator: "Exists"
+    effect: "NoSchedule"
+
+replicas: 2
+
+# grpc support
+grpc: false
+
+# https termination by dex itself
+https: false
+
+ports:
+  web:
+    containerPort: 5556
+    servicePort: 32000
+  # grpc:
+    # containerPort: 5000
+    # servicePort: 35000
+
+service:
+  type: ClusterIP
+
+ingress:
+  enabled: true
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: '/$2'
+    nginx.ingress.kubernetes.io/use-regex: "true"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTP"
+    kubernetes.io/ingress.class: "nginx-control-plane"
+  path: /dex(/|$)(.*)
+  hosts:
+    - null
+  tls: []
```

Comment:
I don't think not setting one will work. The ingress will get a new cert at every start (which is another issue, but anyway), so the cert registered with clients will never be correct, isn't it?